src/main/groovy/life/qbic/portal/sampletracking/DependencyManager.groovy

```diff
@@ -62,7 +62,7 @@ class DependencyManager {
         // set up openBIS connection and data management system object
         def userID = "not logged in"
         try {
-            userID = PortalUtils.getScreenName() ?: configManager.getDataSourceUser()
+            userID = PortalUtils.getScreenName()
```

Comment:
Is there a reason why this was changed back? 
```suggestion
            userID = PortalUtils.getScreenName() ?: configManager.getDataSourceUser()
```

Reply:
Yes because it should not be in there in the first place. Otherwise all users on the portal would have admin access. That is not the desired behaviour

Reply:
Ah makes sense. Thanks for clarifying 👍